alcs-frontend/src/app/features/notice-of-intent/overview/overview.component.ts

```diff
@@ -0,0 +1,98 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject, takeUntil, tap } from 'rxjs';
+import { NoticeOfIntentDetailService } from '../../../services/notice-of-intent/notice-of-intent-detail.service';
+import { NoticeOfIntentDto } from '../../../services/notice-of-intent/notice-of-intent.dto';
+import { TimelineEvent } from '../../../shared/timeline/timeline.component';
+
+const SORTING_ORDER = {
+  //high comes first, 1 shows at bottom
+  MODIFICATION_REVIEW: 12,
+  MODIFICATION_REQUEST: 11,
+  RECON_REVIEW: 10,
+  RECON_REQUEST: 9,
+  CHAIR_REVIEW_DECISION: 8,
+  AUDITED_DECISION: 7,
+  DECISION_MADE: 6,
+  VISIT_REPORTS: 5,
+  VISIT_REQUESTS: 4,
+  ACKNOWLEDGE_COMPLETE: 3,
+  ACKNOWLEDGED_INCOMPLETE: 2,
+  SUBMITTED: 1,
+};
+
+@Component({
+  selector: 'app-overview',
+  templateUrl: './overview.component.html',
+  styleUrls: ['./overview.component.scss'],
+})
+export class OverviewComponent implements OnInit, OnDestroy {
+  $destroy = new Subject<void>();
+  noticeOfIntent?: NoticeOfIntentDto;
+  events: TimelineEvent[] = [];
+  summary = '';
+
+  constructor(private noticeOfIntentDetailService: NoticeOfIntentDetailService) {}
+
+  ngOnInit(): void {
+    this.noticeOfIntentDetailService.$noticeOfIntent
+      .pipe(takeUntil(this.$destroy))
+      .pipe(
+        tap((noi) => {
+          if (noi) {
+            //Load other stuff here
```

Comment:
I guess this is a todo for the next ticket?

Reply:
Yes, this will be needed eventually for loading other things that will go in the timeline